core/storage/base_model/gae_models.py

```diff
@@ -695,6 +695,10 @@ class VersionedModel(BaseModel):
     # The class designated as the snapshot content model. This should be a
     # subclass of BaseSnapshotContentModel.
     SNAPSHOT_CONTENT_CLASS = None
+    # The class designated as the commit log entry model. This should be
+    # a subclass of BaseCommitLogEntryModel. In cases where we do not need
+    # to log the commits it can be None.
+    COMMIT_LOG_ENTRY_CLASS = None
```

Comment:
Is this a regression--wasn't this defined before?

Reply:
Not sure what you mean.